packages/icons/icons.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Meta } from "@storybook/react";
+import string from "devtools/packages/devtools-reps/reps/string";
+import { useState } from "react";
+
+import * as icons from "./index";
+
+export default {
+  title: "Icons",
+} as Meta;
+
+export function BasicUsage() {
+  const [filterValue, setFilterValue] = useState("");
+  return (
+    <div style={{ display: "flex", flexDirection: "column", gap: "2rem" }}>
+      {/* <input
+        placeholder="Filter icons..."
+        value={filterValue}
+        onChange={event => setFilterValue(event.target.value)}
+        style={{ padding: "0.25rem 0.5rem", border: "1px solid grey" }}
+      /> */}
```

Comment:
Did you meant to leave this block commented out? Or did you mean to delete it and the `filterValue` state above?

Reply:
Oops, yeah I'll get this hooked up! Was trying to push it out too quick last night 😅